qiita_pet/static/js/sharing.js

```diff
@@ -1,4 +1,4 @@
-var current_study = null;
+var current_id = null;
```

Comment:
All global variables are possible to remove if you want to - please check the link I sent above and remove them.